settings.gradle.kts

```diff
@@ -20,5 +20,5 @@ dependencyResolutionManagement {
     }
 }
 
-rootProject.name = "Biometrics-SimFace"
-include(":Biometrics-SimFace")
+rootProject.name = "simface"
+include(":simq")
```

Comment:
I have a suspicion that this removes the "simface" from the project; both should be included.

I would suggest moving all of the simFace-related stuff into a subfolder first, tho.